modules/doc/content/examples/ex06_transient.md

```diff
@@ -2,6 +2,49 @@
 
 ## Problem Statement
 
+We consider the transient diffusion equation on the 3D domain $$\Omega$$: find $u$ such that
```

Comment:
Are you trying to style the literal word 'Omega', or create the symbol?
Style:
$Omega$

Symbol:
$\Omega$

If not, this renders a literal '\' in front of Omega. Is this intentional?